transport/grpc_test.go

```diff
@@ -183,8 +217,13 @@ func newGRPCTestEnv(
 	caller string,
 	numInbounds int,
 	procedures []transport.Procedure,
+	maxResponseSize int,
 ) (_ *grpcTestEnv, err error) {
-	yarpcTransport := grpc.NewTransport()
+	var options []grpc.TransportOption
+	if maxResponseSize > 0 {
+		options = append(options, grpc.ServerMaxSendMsgSize(maxResponseSize))
+	}
```

Comment:
why not always set this to a larger size? for the test, right now we're tweaking 2 parameters together, so for the large size, we're tweaking max response size on the client + server max send size on the server. Ideally we only change one of the variables (client side max response size) instead of both for the test

Reply:
Makes sense, I have set it to 10MB